editor.planx.uk/src/pages/FlowEditor/lib/store/editor.ts

```diff
@@ -461,4 +471,20 @@ export const editorStore: StateCreator<
     const inverseOps: OT.Op[] = type.invert(ops);
     send(inverseOps);
   },
+
+  orderedFlow: undefined,
+
+  setOrderedFlow: () => {
```

Comment:
Were we not ordering / sorting the flow already? 

I tried to follow the sortFlow() fn in planx-core to understand the logic, but got a bit caught up in it all.

Reply:
No, the flow is not stored in a sorted / ordered state.

It's an object with keys ordered alphabetically, and the order of traversal is derived from the linked sequence of edges beginning at `_root`. I have a few questions about how this is handled so we'll cover this a little later at the dev call 👍 

Storing the flow sorted would avoid this step being necessary, but would be more complex when it comes up updating and caching the flow.